src/main/core/Subscriber/SecurityEventSubscriber.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace Claroline\CoreBundle\Subscriber;
+
+use Claroline\CoreBundle\Entity\Log\LogSecurity;
+use Claroline\CoreBundle\Event\CatalogEvents\SecurityEvents;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpFoundation\RequestStack;
+use Symfony\Component\Security\Core\Security;
+use Symfony\Contracts\EventDispatcher\Event;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+class SecurityEventSubscriber implements EventSubscriberInterface
+{
+    private $em;
+    private $client;
+    private $security;
+    private $requestStack;
+    private $translator;
+
+    public function __construct(
+        EntityManagerInterface $em,
+        HttpClientInterface $client,
+        Security $security,
+        RequestStack $requestStack,
+        TranslatorInterface $translator
+    ) {
+        $this->em = $em;
+        $this->client = $client;
+        $this->security = $security;
+        $this->requestStack = $requestStack;
+        $this->translator = $translator;
+    }
+
+    public static function getSubscribedEvents(): array
+    {
+        return [
+            SecurityEvents::USER_LOGIN => 'logEvent',
+            SecurityEvents::USER_LOGOUT => 'logEvent',
+            SecurityEvents::USER_DISABLE => 'logEvent',
+            SecurityEvents::USER_ENABLE => 'logEvent',
+            SecurityEvents::NEW_PASSWORD => 'logEvent',
+            SecurityEvents::FORGOT_PASSWORD => 'logEvent',
+            SecurityEvents::ADD_ROLE => 'logEvent',
+            SecurityEvents::REMOVE_ROLE => 'logEvent',
+            SecurityEvents::VIEW_AS => 'logEvent',
+        ];
+    }
+
+    public function logEvent(Event $event, string $eventName): void
+    {
+        $request = $this->requestStack->getCurrentRequest();
+
+        $logEntry = new LogSecurity();
+        $logEntry->setDetails(sprintf('Log %s: %s', $eventName, $event->getMessage($this->translator)));
+        $logEntry->setEvent($eventName);
+        $logEntry->setTarget($event->getUser());
+        $logEntry->setDoer($this->security->getUser());
+        $logEntry->setDoerIp($request->getClientIp());
```

Comment:
Attention, stocker une IP c'est compliqué de nos jours avec les lois RGPD.
Je pense qu'on peux utiliser ça: https://symfony.com/blog/new-in-symfony-4-4-ip-address-anonymizer

> A common procedure to store information about users (for analysis purposes) while being compliant is to anonymize IP addresses. Specifically, you must remove the last byte for IPv4 addresses and the last 8 bytes for IPv6 addresses.

La donnée reste utilisable pour faire du match d'ip et à priori ça nous évite d'avoir a demander à l'utilisateur de consentir avec une checkbox unchecked par defaut. 
A moins qu'on ai déjà tout ça ? ping @LaurentGruber @Elorfin

Reply:
Change done, thx :)